src/frontend/data/websocket.ts

```diff
@@ -0,0 +1,29 @@
+import RobustWebSocket from 'altamoon-robust-websocket';
+
+import { addResource } from 'data/stores/generics';
+import { UploadableObject, Video } from 'types/tracks';
+import { WS_ENPOINT } from 'settings';
+import { modelName } from 'types/models';
+
+import { appData } from './appData';
+
+type WSMessageType = {
+  resource: UploadableObject;
+  type: modelName;
+};
+
+let videoWebsocket: RobustWebSocket;
+
+export const initVideoWebsocket = (video: Video) => {
+  if (videoWebsocket) return;
+  const location = window.location;
+  const wsProto = location.protocol === 'https' ? 'wss' : 'ws';
+  videoWebsocket = new RobustWebSocket(
```

Comment:
```suggestion
  videoWebsocket = new RobustWebSocket(
```
Is this can generate error? I see in the documentation that we can set timeout or automatically reconnect, none of this options are interesting for us ?

Reply:
We are using the default behaviour out of the box The lib automatically reconnect to the server when the websocket is disconnected.
All you see in the documentation is about changing all this default value/behaviour I think.